lib/bindings/http/validation/structured.js

```diff
@@ -14,8 +14,7 @@ function check(payload, headers, receiver) {
   const sanityHeaders = sanityAndClone(headers);
 
   // Validation Level 1
-  if (!receiver.allowedContentTypes
-    .includes(sanityHeaders[HEADER_CONTENT_TYPE])) {
+  if (!receiver.allowedContentTypes.includes(sanityHeaders[HEADER_CONTENT_TYPE])) {
```

Comment:
Not a blocker, but I remember something like that in other file (?) : ) 

if ( not_receiver_allowed_includes foo )

Reply:
Sorry missing the context here, I remember that we saw something like that in other file.
and you extracted some variables to improve the readability.

Reply:
@lance I found it https://github.com/cloudevents/sdk-javascript/pull/134#discussion_r422346076

Reply:
Ahh gotcha. The difference here is that this is for a structured receiver. The PR you point to is modifying a binary event receiver where an event is sent without any content-type header. For structured receivers, [the content-type must be there](https://github.com/cloudevents/spec/blob/v1.0/http-protocol-binding.md#321-http-content-type).